Error while checking build/O2Physics/o2 for 967f79998d9cd79ef7ea661eafbfd1bedbc18474 at 2026-04-28 16:24:
```
## sw/BUILD/O2Physics-latest/log
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGUD/Tasks/flowCorrelationsUpc.cxx:378:78: error: unused parameter 'phi' [-Werror=unused-parameter]
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGUD/Tasks/flowCorrelationsUpc.cxx:378:89: error: unused parameter 'eta' [-Werror=unused-parameter]
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGUD/Tasks/flowCorrelationsUpc.cxx:378:110: error: unused parameter 'vtxz' [-Werror=unused-parameter]
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGUD/Tasks/flowCorrelationsUpc.cxx:394:60: error: unused parameter 'runNumber' [-Werror=unused-parameter]
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/16001/967f79998d9cd79ef7ea661eafbfd1bedbc18474/build_O2Physics_o2/pretty.html).

--- a/PWGUD/Tasks/flowCorrelationsUpc.cxx
+++ b/PWGUD/Tasks/flowCorrelationsUpc.cxx
@@ -375,7 +375,7 @@ struct FlowCorrelationsUpc {
     return true;
   }
 
-  bool setCurrentParticleWeights(float& weight_nue, float& weight_nua, float phi, float eta, float pt, float vtxz)
+  bool setCurrentParticleWeights(float& weight_nue, float& weight_nua)
   {
     float eff = 1.;
     if (mEfficiency)
@@ -391,7 +391,7 @@ struct FlowCorrelationsUpc {
 
   // fill multiple histograms
   template <typename TCollision, typename TTracks>
-  void fillYield(TCollision collision, TTracks tracks, int runNumber, float vtxz) // function to fill the yield and etaphi histograms.
+  void fillYield(TCollision collision, TTracks tracks, float vtxz) // function to fill the yield and etaphi histograms.
   {
     registry.fill(HIST("Nch"), tracks.size());
     registry.fill(HIST("zVtx"), collision.posZ());

--- a/PWGUD/Tasks/flowCorrelationsUpc.cxx
+++ b/PWGUD/Tasks/flowCorrelationsUpc.cxx
@@ -375,7 +375,7 @@ struct FlowCorrelationsUpc {
     return true;
   }
 
-  bool setCurrentParticleWeights(float& weight_nue, float& weight_nua)
+  bool setCurrentParticleWeights(float& weight_nue, float& weight_nua, float pt)
   {
     float eff = 1.;
     if (mEfficiency)
@@ -446,7 +446,7 @@ struct FlowCorrelationsUpc {
 
       // 计算track1的权重
       float weff1 = 1., wacc1 = 1.;
-      if (!setCurrentParticleWeights(weff1, wacc1, phi1, eta1, pt1, vtxz))
+      if (!setCurrentParticleWeights(weff1, wacc1, pt1))
         continue;
 
       if (system == SameEvent) {
@@ -556,7 +556,7 @@ struct FlowCorrelationsUpc {
       independent = nTracksCorrected;
     }
 
-    fillYield(collision, tracks, currentRunNumber, vtxz);
+    fillYield(collision, tracks, vtxz);
 
     fillCorrelations<CorrelationContainer::kCFStepReconstructed>(
       tracks, tracks, collision.posZ(), SameEvent,

